### PR TITLE
Birth correction on mobile test, add e2e mobile helpers

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Core: https://github.com/opencrvs/opencrvs-core/pull/10746

* Add test for birth correction flow on mobile
* Add helper functions for mobile testing:
  * `setMobileViewport()`
  * `isMobile()`
* Add support to `selectAction()` for mobile
* Add `navigateToWorkqueue()` helper

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/v2-utils.ts [hunk 1]
```diff
@@ -26,23 +48,11 @@ export async function selectAction(
     await ensureAssigned(page)
   }
 
-  // Keep retrying the click until the dropdown is visible
-  let isVisible = false
-  let attempts = 0
-  const maxAttempts = 10
-
-  while (!isVisible && attempts < maxAttempts) {
```
Comment (cibelius) [inline e2e/v2-utils.ts:34]: This works fine without the while loop, tested here: https://github.com/opencrvs/e2e/actions/runs/18525107550

**Review by jamil314 — APPROVED** [review]
Great one 👑
